lib/polyn/serializers/json.rb

```diff
@@ -129,7 +129,8 @@ def self.combined_error_message(event, errors)
       end
 
       def self.store_name(**opts)
-        opts.fetch(:store_name, Polyn::SchemaStore.store_name)
+        name = opts.fetch(:store_name, Polyn::SchemaStore.store_name)
+        name || Polyn::SchemaStore.store_name
```

Comment:
why not just call `opts[:store_name] || Polyn::SchemaStore.store_name` as a one liner here?